x/coinmaster/keeper/params.go

```diff
@@ -14,8 +18,34 @@ func (k Keeper) GetParams(ctx sdk.Context) types.Params {
 }
 
 // SetParams set the params
-func (k Keeper) SetParams(ctx sdk.Context, params types.Params) {
+func (k Keeper) SetParams(ctx sdk.Context, params types.Params) (err error) {
+	if params.Minters != "" {
+		// validate that params.Minters is a comma separated list of addresses
+		mintersRegex := (`^([a-z0-9]{40,80},)*[a-z0-9]{40,80}$`)
```

Comment:
Let's throw the regex into a utility function, something like "splitAddresses" and then we can surround that with a bunch of edge-case and happy case tests.

I also wonder if there's a utility function out there that splits comma-separated addresses for us already?

Reply:
Good idea: refactored in 825a400a26d021b5300c52e382ddab29babcd231